src/averageAges.js

```diff
@@ -1,64 +1,57 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+const FEMALE = 'f';
+const MALE = 'm';
+
+function sumAges(whos) {
+  return whos.reduce((accum, person) =>
+    (accum + (person['died'] - person['born'])), 0);
```

Comment:
you can change that function to calculating avgAge by adding `/ whos.length` after reduce so you don't need to divide by length later in your code